docs/Deploying-Pega-on-GKE.md

```diff
@@ -370,7 +333,64 @@ $ kubectl create namespace pegaaddons
 namespace/pegaaddons created
 ```
 
-15. Install the addons chart, which you updated in [Preparing your local system](#preparing-your-gke-resources--45-minutes).
+12. (Optional:) To support HTTPS connectivity with Pega Platform, do one of the following:
+
+- To support HTTPS connectivity with Pega Platform using a secret:
+
+  a. To must pass the appropriate certificate you created, enter:
+
+`kubectl create secret tls <secret-name> --cert <cert.crt-file> --key <private.key-file> --namespace <namespace-name>`
+
+For HTTPS support using a secrets file, ensure that you make the following changes in the pega.yaml file for the exposed tiers in your deployment:
+
+```yaml
+ingress:
+        domain: "web.dev.pega.io"
+        tls:
+          enabled: true
+          secretName: web-domain-certificate
+          useManagedCertificate: false
+          ssl_annotation:
+```
+
+    b. To ensure the certificate is working in the cluster, enter:
+
+`kubectl get secrets --namespace <namespace-name>`
+
+-  To support HTTPS connectivity with Pega Platform using a pre-shared certificate, you must upload the pre-shared certificate your Google Cloud project and update the pega.yaml file appropriately.
+
+   To upload the appropriate certificate your Google Cloud project, enter:
+
+`gcloud compute ssl-certificates create test-ingress-1 --certificate [FIRST_CERT_FILE] --private-key [FIRST_KEY_FILE]`
+
+For HTTPS support using a pre-shared certificate, ensure that you make the following changes in the pega.yaml file for the exposed tiers in your deployment:
+
+```yaml
+ingress:
+        domain: "web.dev.pega.io"
+        tls:
+          enabled: true
+          secretName:
+          useManagedCertificate: false
+          ssl_annotation:
+             ingress.gcp.kubernetes.io/pre-shared-cert: webCert
+```
+
+- To support HTTPS connectivity with Pega Platform, you configure a managed SSL certificate and associate it with the ingress by making the following changes in the pega.yaml file for the exposed tiers in your deployment:
+
+```yaml
+ingress:
```

Comment:
This needs to change.  For managed certificate, the following yaml stub should be used :

ingress:
          domain: "webtest.gketest1.dev.pega.io"
          tls:
            enabled: true
            secretName:
            useManagedCertificate: true
            ssl_annotation:
              # This is not mandatory.  We can create a static IP address and use it in any of the approaches using this annotation.
              kubernetes.io/ingress.global-static-ip-name: lb-ip-address

Reply:
updated the step and incorporated your review comments